website/versioned_docs/version-5.13.0/cdm-materials.md

```diff
@@ -0,0 +1,64 @@
+
+
+## CDM RESOURCES
+### [A discussion of "The Future of the Securities Lending Market: An Agenda for Change"](https://www.finos.org/blog/the-future-of-securities-lending-market)
+
+ISLA published a paper entitled ‘The Future of the Securities Lending Market | An Agenda for Change’ in September 2019, looking at the conjunction of market complexity, technical architecture and regulatory impacts. The conclusion reached was that a transformation was overdue and the solution would be underpinned by common standards.
+
+### [CDM Now Available In Python](https://www.finos.org/blog/cdm-now-available-in-python)
+
+A recent contribution to the CDM project makes it available in the Python programming language for the first time, which will allow a new community of developers to use the CDM in their projects.
+
+### [Common Domain Model into Open Source at FINOS](https://www.finos.org/hubfs/Accepted%20FINOS%20Edits_%20Common%20Domain%20Model%20into%20Open%20Source%20at%20FINOS.docx.pdf)
+
+Read more about CDM's journey into open source, project contribution guidelines, principles, and frequently asked questions.
+
+### [Barclay's demonstration of interest rate swap processing using the Common Domain Model (CDM)](https://www.youtube.com/watch?v=LTWn_AWC4F8)
+
+In this recording, Barclays introduces a prototype for processing interest rate swaps using the common domain model. Starting with how the post-trade industry in capital markets has evolved over time, resulting in procedural inefficiencies that are complex, duplicative, and costly to maintain. The common domain model covers processes, events, and data, and Barclays' prototype aims to showcase the potential of this new standard in revolutionizing the post-trade industry. 
+
+### [ISLA, ICMA & ISDA Appoint FINOS for CDM Repository](https://www.islaemea.org/press-releases/isla-icma-isda-appoint-finos-for-cdm-repository/)
+
+On Thursday, September 8th, 2022, three leading organizations in the financial industry, International Securities Lending Association (ISLA), International Capital Market Association (ICMA), and International Swaps and Derivatives Association (ISDA), made a major announcement appointing FINOS to provide a repository for the Common Domain Model (CDM) with the aim of fostering growth in the open-source community.
+
+### [2022 Open Source in Finance Forum New York Keynote Panel: Common Domain Model (CDM)](https://resources.finos.org/content/keynote-panel-common-domain-model-cdm/?projects-sigs=common-domain-model-cdm)
+
+At the keynote session of the 2022 Open Source in Finance Forum New York, a panel of industry leaders including Adrian Dale of ISLA, Allan Milligan of ISDA, and Georgina Jarrett, joined moderator Ffion Acclan of Goldman Sachs to delve into the exciting topic of trade association collaboration, open sourcing, and the creation of a common domain for the market, built by the market. 
```

Comment:
Alan again
Actually it's Ffion Acland - same will be wrong on the previous file too